engine/render/src/render/render_script.cpp

```diff
@@ -1653,6 +1683,8 @@ namespace dmRender
      * ```lua
      * -- get the width of the render target color buffer
      * local w = render.get_render_target_width(self.target_right, render.BUFFER_COLOR_BIT)
+     * -- get the width of a render target resource
+     * local w = render.get_render_target_width('my_rt_resource', render.BUFFER_COLOR_BIT)
```

Comment:
For future us: Next time, I think we should return a 2-tuple, as that's supported by Lua.
Feels a bit strange to do all the steps to get first width, then height in 2 function calls.

Reply:
yes i agree, we should deprecate these functions. in the graphics adapter we get both width and height from the same call, so it's a bit weird to have this separation